docs/reference/secrets-reference.md

```diff
@@ -0,0 +1,42 @@
+---
+id: secrets-reference
+title: Secrets YAML Reference
+---
+Recyclarr supports secret value substitution within it's YAML configuration files. This keeps sensitive
+values out of configuration files and makes it easier to share configurations without having to
+perform manual redaction. Using secrets is optional and support was introduced in version `TBD`.
+
+The `secrets.yml` file is located in the following locations (by default) depending on your
+platform:
+
+| Platform | Location                                               |
+| -------- | ------------------------------------------------------ |
+| Windows  | `%APPDATA%\recyclarr\secrets.yml`                     |
+| Linux    | `~/.config/recyclarr/secrets.yml`                     |
+| MacOS    | `~/Library/Application Support/recyclarr/secrets.yml` |
```

Comment:
Let's not repeat the application data directory paths again here. I realize there might be places where I've already duplicated this, but my intent is to remove those duplicates eventually.

Reason: The paths may change in the future, or need other updates, which is easier to do if it's in one place.

Solution: Link to the aplication data directory page instead. I typically use this link:

```
/file-structure.md#appdata-directory
```